boxoffice/templates/order.html

```diff
@@ -139,4 +139,5 @@ <h4 class="text-center attendee-form-title">Please fill the attendee details</h4
   </div>
 {% endblock %}
 {% block footerscripts %}
+  <script src='/static/js/views/order.js'></script>
```

Comment:
In this case, then in file boxoffice/views/__init__.py
js assets will have to be updated.
assets['boxoffice.js'][version] = 'js/scripts.js'